src/EnergyPlus/DataEnvironment.hh

```diff
@@ -22,6 +22,7 @@ namespace DataEnvironment {
 	extern Real64 const EarthRadius; // Radius of the Earth (m)
 	extern Real64 const AtmosphericTempGradient; // Standard atmospheric air temperature gradient (K/m)
 	extern Real64 const SunIsUpValue; // if Cos Zenith Angle of the sun is >= this value, the sun is "up"
+	extern Real64 const StdBaroPressSeaLevel; // Standard barometric pressure at sea level (Pa)
```

Comment:
I happen to have just drafted out similar changes and came up with different names and to also add to weather manager a one time calculation of rho at STP.

	extern Real64 const stdAtmosphericPressure; // one atmosphere, sea level standard pressure 101325 Pa
	extern Real64 rhoAirSTP; // Standard density of dry air at 101325 Pa, 20.0C temperaure